NEWS

```diff
@@ -1,3 +1,65 @@
+KLEE 2.2, xx November 2020
+==========================
+
+Incorporating changes from 4 March 2020 to xx November 2020.
+Maintainers during this time span: @ccadar and @MartinNowack
+Documentation at http://klee.github.io/releases/docs/v2.2
+
+== Major features ==
+- Added support for C++ exceptions (@corrodedHash, @jbuening, @futile)
+- Correctly copy variadic arguments with byval attribute (@ccadar)
+- Modified the random path searcher to work on a subset of states (@kren1)
+- Added state IDs to improve determinism (@251)
+- Overhauled the (Travis CI) build scripts (@MartinNowack)
+- Restructured header files (@ccadar)
+
+== LLVM support ==
+- Current recommended version is LLVM 9
+- We have decided to extend support for LLVM <3.8, but KLEE 2.2 will be the last version with support for LLVM <6
+- Added partial support for LLVM 11 (@lzaoral)
+
+== Options, scripts and KLEE intrinsics added, changed or removed ==
+- Changed --debug-print-instructions to also print state IDs (@251)
+- Added -rng-initial-seed to set the seed for KLEE's random number generator (@251)
+- Added klee_is_replay intrinsic which returns whether the code is being executed symbolically or in replay mode (@alastairreid)
+- Added --compress-process-tree to remove intermediate nodes in the process tree when possible (@sebastianpoeplau)
+- Added klee-zesti, a ZESTI-like wrapper (@kren1)
+- Added --table-format=readable-csv/csv to klee-stats (@251)
+
+== Other changes ==
+- Fixed GlobalAlias initialization (@jbuening)
+- Enforced fork/branch limits in branch() and fix double termination (@251)
+- Enhanced POSIX runtime in the case where a symbolic file is given as an absolute path, i.e. /current/work/dir/A (@kren1)
+- Named jobs in Travis CI for better visualization of results (@ccadar)
+- Fixes and improvements in the statistics code including klee-stats (@251)
+- Added a simple model for GET/SET_LK (@kren1)
+- Fixed bug in the handling of vectorized code (@Warfley)
+- Fixed bug in the handiling of STP array names (@MartinNowack)
```

Comment:
Typo

Reply:
Thanks -- and this is after spellchecking it! ;)